src/App.js

```diff
@@ -1,20 +1,22 @@
 import { Routes, Route } from 'react-router-dom';
+import { Provider } from 'react-redux';
 import Header from './components/Header';
 import Rockets from './pages/Rockets';
 import Missions from './pages/Missions';
 import Profile from './pages/Profile';
 import Dragons from './pages/Dragons';
+import elementStore from './redux/configureStore';
 
 const App = () => (
-  <>
+  <Provider store={elementStore}>
     <Header />
     <Routes>
       <Route exact path="/" element={<Rockets />} />
       <Route path="/missions" element={<Missions />} />
       <Route path="/profile" element={<Profile />} />
       <Route path="/dragons" element={<Dragons />} />
     </Routes>
-  </>
+  </Provider>
```

Comment:
Good effort on using the adding the `Provider` with the `store`. However, It is recommended that you wrap the `App` component with `Provider` inside the `index.js` file instead of including it inside the `App` component.

For Example:  inside the `index.js` file 
```
 import React from 'react';
import ReactDOM from 'react-dom';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import store from './redux/configureStore';
import './index.scss';
import App from './App';

ReactDOM.render(
  <React.StrictMode>
    <BrowserRouter>
      <Provider store={store}>
        <App />
      </Provider>
    </BrowserRouter>
  </React.StrictMode>,
  document.getElementById('root'),
);

// If you want to start measuring performance in your app, pass a function
// to log results (for example: reportWebVitals(console.log))
// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
```